app/src/main/java/ch/nevis/exampleapp/ui/selectAuthenticator/model/AuthenticatorItem.kt

```diff
@@ -0,0 +1,54 @@
+package ch.nevis.exampleapp.ui.selectAuthenticator.model
+
+import androidx.annotation.StringRes
+import ch.nevis.exampleapp.R
+import ch.nevis.mobile.sdk.api.localdata.Authenticator
+
+/**
+ * Represents an authenticator that is listed and can be selected by the user on select authenticator view.
+ *
+ * @throws IllegalArgumentException when an unknown AAID is used.
+ */
+data class AuthenticatorItem(
+    /**
+     * The AAID of the authenticator.
+     */
+    val aaid: String,
+
+    /**
+     * The flag that tells whether the authenticator is server policy compliant or not.
+     */
+    val isPolicyCompliant: Boolean,
+
+    /**
+     * The flag that tells whether the user already enrolled the authenticator or not.
+     */
+    val isUserEnrolled: Boolean
+) {
+    /**
+     * String resource identifier of the title of the authenticator.
+     */
+    @StringRes
+    val titleResId = authenticatorTitles[aaid] ?: throw IllegalArgumentException("Unknown AAID.")
```

Comment:
I'm not sure it is the best solution to throw an `IllegalArgumentException` in this case, but once again It should never happen.